src/snakemake_interface_logger_plugins/settings.py

```
@@ -5,18 +5,21 @@
 __email__ = "johannes.koester@uni-due.de"
 __license__ = "MIT"
 
+from abc import ABC, abstractmethod
```

Ruff (0.8.2)

`abc.ABC` imported but unused

Remove unused import

(F401)